LongWarOfTheChosen/Src/LW_Overhaul/Classes/UIPersonnel_SoldierListItem_LW.uc

```diff
@@ -302,32 +311,32 @@ function AddNameColumnIcons(XComGameState_Unit Unit)
 		{
 			switch (Unit.ComInt)
 			{
-			case eComInt_Standard:
-				comIntIconPath = "Standard";
-				ComIntColor = "0xbf1e2e";
-				break;
-			case eComInt_AboveAverage:
-				comIntIconPath = "AboveAverage";
-				ComIntColor = "0xe69831";
-				break;
-			case eComInt_Gifted:
-				comIntIconPath = "Gifted";
-				ComIntColor = "0xfdce2b";
-				break;
-			case eComInt_Genius:
-				comIntIconPath = "Genius";
-				ComIntColor = "0x53b45e";
-				break;
-			case eComInt_Savant:
-				comIntIconPath = "Savant";
-				ComIntColor = "0x27aae1";
-				break;
+				case eComInt_Standard:
+					comIntIconPath = "Standard";
+					ComIntColor = "0xbf1e2e";
+					break;
+				case eComInt_AboveAverage:
+					comIntIconPath = "AboveAverage";
+					ComIntColor = "0xe69831";
+					break;
+				case eComInt_Gifted:
+					comIntIconPath = "Gifted";
+					ComIntColor = "0xfdce2b";
+					break;
+				case eComInt_Genius:
+					comIntIconPath = "Genius";
+					ComIntColor = "0x53b45e";
+					break;
+				case eComInt_Savant:
+					comIntIconPath = "Savant";
+					ComIntColor = "0x27aae1";
+					break;
```

Comment:
I prefer `case` aligned with `switch` to avoid that extra level of indenting. It's inconsistent in the code, but I would just leave things as they are in the case of `switch` statements.